infrastructure/variables.tf

```diff
@@ -45,3 +45,10 @@ variable "database_sku_capacity" {
   default = "8"
 }
 
+variable "business_area" {
+  default = "cft"
+}
```

Comment:
Any ideas what this is for?  I can see it being used in the DB config so sounds like it is correct.